examples/Polygons.js

```diff
@@ -145,7 +145,7 @@ requirejs(['./WorldWindShim',
         ];
 
         polygonAttributes = new WorldWind.ShapeAttributes(null);
-        polygonAttributes.imageSource = "../images/400x230-splash-nww.png";
+        polygonAttributes.imageSource = "../examples/data/400x230-splash-nww.png";
```

Comment:
I mentioned in the issue the full path of the file, but using the relative "data/400x230-splash-nww.png" suffices, as it's seen in lines 106, 205, etc.